appveyor.yml

```diff
@@ -2,6 +2,8 @@
 
 environment:
   matrix:
```

Comment:
can we add `DEBUG_RESOLVER=1` to the environment?

That should show us a whole ton of output for the dependency resolution